specification/components/schemas/Patient.yaml

```diff
@@ -1,4 +1,9 @@
 type: object
+required:
+  - birthDate
+  - gender
+  - name
+  - address
```

Comment:
is id (nhsNumber) not required as well?

Reply:
Yes - omitted because it will not be required for allocation